environ/environ.py

```diff
@@ -18,15 +18,9 @@
 import sys
 import urllib.parse as urlparselib
 import warnings
-from urllib.parse import (
-    parse_qs,
-    ParseResult,
-    unquote_plus,
-    urlparse,
-    urlunparse,
-)
+from urllib.parse import ParseResult, parse_qs, unquote_plus, urlparse, urlunparse
```

Comment:
Imported names are in the wrong order
```suggestion
from urllib.parse import parse_qs, ParseResult, unquote_plus, urlparse, urlunparse
```

Reply:
Also this line too long (82 > 79 characters)